src/main/java/org/takes/facets/slf4j/TsLogged.java

```diff
@@ -25,6 +25,7 @@
 package org.takes.facets.slf4j;
 
 import java.io.IOException;
+import lombok.EqualsAndHashCode;
```

Comment:
@dmzaytsev let's remove the blank line between the copyright and `package` here also.